README.md

```diff
@@ -1,2 +1,25 @@
 # zap-cli
-A Node.js CLI command for searching for, or opening a YouTube video with Zen Audio Player.
+
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zen-audio-player/zen-audio-player.github.io?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Build Status]https://api.travis-ci.org/zen-audio-player/zap-cli.svg?branch=master)](https://travis-ci.org/zen-audio-player/zap-cli)
+[![GitHub Stats](https://img.shields.io/badge/github-stats-ff5500.svg)](http://githubstats.com/zen-audio-player/zap-cli)
+
+A Node.js CLI tool for searching for, or opening a YouTube video with [Zen Audio Player](https://github.com/zen-audio-player/zen-audio-player.github.io).
+
+## Usage
+To use the zap-cli command-line tool, clone the repository and run a global npm install inside of it.
+
+```shell
+git clone https://github.com/zen-audio-player/zap-cli 
+cd zap-cli
+npm install -g
```

Comment:
Does this need to be `npm install -g .`? Also, what about this project's dependencies, do they get pulled in automatically by that command?

If I were doing this explicitly, I would do

```script
git clone https://github.com/zen-audio-player/zap-cli 
cd zap-cli
npm install 
npm install -g .
```

Please manually verify the steps (ie: delete any local clones and start from scratch to make sure the steps work).